pages/src/highscore.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```

Comment:
high scores is also in the handlebars...it needs to reference the api/scores though.